Source/AliveLibAE/MainMenu.cpp

```diff
@@ -3001,33 +3074,35 @@ void MainMenuController::HandleMainMenuUpdate()
         }
 
         // Todo: change all field_10_fn_update function types to unsigned.
-        const unsigned int pageUpdateRet = (this->*(pPage->field_10_fn_update))(inputHeld);
+        const unsigned int pageUpdateReturnedCam = (this->*(pPage->field_10_fn_update))(inputHeld);
```

Comment:
you could probably change these into a struct, something like:

struct NextPage
{
BYTE targetCamera;
BYTE screenChangeEffect
};

?

Then you won't need the >> 16 & 0xFF and the BYTE1() on the return value

Reply:
yup that will be in the next PR, along with the function and lambda return values